Allow query param passthrough for paginate links

KronicDeth opened the pull request:
# Changelog
## Bug Fixes
* Allow query params to passthrough for paginate links

KronicDeth commented:
I'm not sure what `coveralls.io` doesn't like the build.  I matched the number it gave exactly and it still shows a red build X.

jeffutter commented:
Hey sorry, this took me so long to get to. Busy week followed by vacation followed by busy week :)

This looks good I think. Just so I know what's coming, this will be a breaking change because it removes the `subject` field, from the index/show/etc methods, yes?

Do you have an example of what the user-facing code updates would look like?

KronicDeth commented:
@jeffutter I updated 
* Calcinator.View` callbacks and docs to reflect the change
* Changed the version number to `6.0.0` to reflect it is a breaking change in the cases where users didn't `use Calcinator.PhoenixView`.
* Added a section to `UPGRADING.md` explaining the change and if/how to adapt (this covers the question you had)
* Moved the `v5.1.0` section of the `CHANGELOG.md` to `v6.0.0` since `v5.1.0` wasn't released to `hex.pm` before this bump of the version number.
